test/objects/nodebalancers/nodes_test.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime
+
+from test.base import ClientBaseCase
+from linode.objects.base import MappedObject
+
+from linode.objects import NodeBalancerConfig, NodeBalancerNode
+
+
+class NodeBalancerNodeTest(ClientBaseCase):
+    """
+    Tests methods of the NodeBalancerNode class
+    """
+    def test_get_node(self):
+        """
+        Tests that a node is loaded correctly by ID
+        """
+        node = NodeBalancerNode(self.client, 54321, (65432, 123456))
+        self.assertEqual(node._populated, False)
+
+        self.assertEqual(node.weight, 50)
+        self.assertEqual(node._populated, True)
+
+        self.assertEqual(node.id, 54321)
+        self.assertEqual(node.address, "192.168.210.120")
+        self.assertEqual(node.label, "node54321")
+        self.assertEqual(node.status, "UP")
+        self.assertEqual(node.mode, "accept")
+        self.assertEqual(node.config_id, 65432)
+        self.assertEqual(node.nodebalancer_id, 123456)
+
+    def test_create_node(self):
+        """
+        Tests that a node can be created
+        """
+        with self.mock_post('nodebalancers/123456/configs/65432/nodes') as m:
```

Comment:
this should be `with self.mock_post('nodebalancers/123456/configs/65432/nodes/54321') as m:` - this POST will return a single node, not a list of them